src/pages/OrderPage.tsx

```diff
@@ -35,6 +35,7 @@ const OrderPage: CustomVFC<{}, { order: hasura.GET_ORDERS_PRODUCT['order_log_by_
   const location = useLocation()
   const history = useHistory()
   const [withTracking] = useQueryParam('tracking', BooleanParam)
+  const [withCode] = useQueryParam('code', StringParam)
```

Comment:
NITS 
normally "with..." present Boolean type
and the code parameter is some value like error code "E_SYNC_PAYMENT"